skeleton/default.nix

```diff
@@ -15,23 +15,72 @@
   }
 }:
 with obelisk;
-project {} ({ pkgs, ... }: {
+let args = {
+  # You must accept the Android Software Development Kit License Agreement at
+  # https://developer.android.com/studio/terms in order to build Android apps.
+  # Uncomment this to indicate your acceptance:
+  android_sdk_accept_license = true;
+  allowUnfree = true;
```

Comment:
Should be set to false before merging. 
How do we make sure CI caches it though? 
Add the accept/allow as args to this file defaulting to false and overriden by CI?